app/controllers/api/v1/document_change_requests_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class Api::V1::DocumentChangeRequestsController < Api::V1::ApplicationController
+  skip_before_action :verify_authenticity_token, only: :update
+  before_action :check_token_and_set_application, only: :update, if: :json_request?
+  before_action :check_file_params_are_present, only: :update, if: :json_request?
```

Comment:
This actually asks questions for the previous work we've done too, but why do we have `if: :json_request?` on these? Surely we *always* want to run these security checks?

Reply:
Good point - I had copied it over from another controller, will take it off